Samples/ResourceManagement/cs-winui/winui_desktop_packaged_app/winui_desktop_packaged_app.csproj

```diff
@@ -21,9 +21,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ProjectReunion" Version="0.8.1" />
-    <PackageReference Include="Microsoft.ProjectReunion.Foundation" Version="0.8.1" />
-    <PackageReference Include="Microsoft.ProjectReunion.WinUI" Version="0.8.1" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0-preview1" />
```

Comment:
I would have expected to also see Microsoft.Windows.SDK.BuildTools listed here... Same for the other CSPROJ and WAPPROJ files. You can see that the package shows up in the VCXPROJ files.

Can you check in NuGet Package Manager whether these projects have Microsoft.Windows.SDK.BuildTools installed?

Reply:
Hmmm interesting... 
If I uninstall and install again Microsoft.WindowsAppSDK, I get this:
```
Successfully installed 'Microsoft.Windows.SDK.BuildTools 10.0.20348.19' to winui_desktop_packaged_app
Successfully installed 'Microsoft.WindowsAppSDK 1.0.0-preview1' to winui_desktop_packaged_app
```
However, I don't see the SDK BuildTools installed after that, and they also don't appear in the csproj files and others. That said, the solution still builds correctly in all platforms and configurations. 

Is it possible that VS determines that it doesn't need the BuildTools? Can that happen?

Reply:
> Is it possible that VS determines that it doesn't need the BuildTools? Can that happen?
AFAIK, no...

Here's my suggestion. Build the app in VS Developer Command Prompt with /bl. Then, search for makepri.exe in https://msbuildlog.com/. See which makepri.exe was used.

Reply:
i think you don't need the explicit reference to buildtools with PackageReference.

Reply:
that makes sense, I do see the makepri.exe of the Buildtools being used, but I just can't seem to see it in packageReference. Do you know why's this, @huichen123 ?